gnuradio-runtime/lib/vmcircbuf.h

```diff
@@ -27,7 +27,7 @@
 #include <gnuradio/thread/thread.h>
 #include <vector>
 
-extern gr::thread::mutex s_vm_mutex;
+extern std::mutex s_vm_mutex;
```

Comment:
Had forgotten about that surprising `extern`; nothing to do with your work here, but let me / us investigate the design here in #2996.